_docs/manage/volumes/index.md

```diff
@@ -129,3 +129,22 @@ default/volume-name
 
 Volumes may not be removed immediately as the data will be purged in the
 background.
+
+## Using node selector
+
+Node selector is a form of constraint. For the volume to be eligible for a deployment on a node, selector must match node's labels. An empty selector
```

Comment:
"A node selector is a constraint on where volumes may be deployed, based on node labels."